tests/conftest.py

```diff
@@ -167,7 +167,7 @@ def __init__(self):
         config_metropolis['BLOCK_GAS_LIMIT'] = 2**60
 
         for a in range(10):
-            tester.base_alloc[getattr(tester, 'a%i' % a)] = {'balance': 10**24}
+            tester.base_alloc[getattr(tester, 'a%i' % a)] = {'balance': 10**40}
```

Comment:
Why the drastic increase in seeded capital?  That is more ETH than there is in existence I believe.

Reply:
I could lower it a bit, but the sum of creatorEthRequiredLong + creatorEthRequiredShort + creatorTokens is typically on the order of 10**36.  This is because the number of clicks is now 10**18.

Reply:
Can we edit the test to not try to buy so many shares?  I'm not sure how it is deciding how many shares to buy, but it seems like buying 10^20 shares is incredibly silly when the shares cost 1 ether each.

Reply:
Dang, tester is a baller

Reply:
@MicahZoltu Like I mentioned on Slack, the number of shares is getting set to a fixed point number, so it ends up being a really high number like 8000000000000000000, and the price gets also gets set to a hight number like 547868758977303680 due to the number of ticks.  I suggested dividing them by 10**18 to make them smaller, but I believe you said the price was valid (albeit silly) and that the error was probably somewhere else.  Dividing by 10^18 seems like a sensible solution.  For example, in the above example, it would result in the number of shares being 8, and the price being $0.54, which seems like reasonable.

Reply:
The price is fine, and should be a number between `[0, 10^18]`.  However, there is no need to have the test try to buy `10^18` shares like it is at the moment if it is doing something like `fix('1')`.  If the price of one share is `10^18`, then the tests should be buying only a few shares, on the order of 1-1000.

Rather than doing `fix(1) / 10^18`, instead we should just do `1` for the number of purchased shares (or similar).  In theory, this should just be a matter of removing the `fix(...)` wrapper around share counts, though I haven't dug deeply into these tests to confirm that.  Such a change would make it so we don't need to give the test accounts 10^22 ETH.

Reply:
Ok, that's fine.  I can remove the call to `fix` when setting the number of shares.  That should be sufficient.